resources/homegrown/data-clinic-for-general.qmd

```diff
@@ -1,76 +1,268 @@
+   - Example 1: A Requestor is conducting a new QIP and would like data on blood gas results performed throughout a patient's ICU stay. This is not covered by existing reports (e.g. ICNARC report) and as it is a new project, we do not have an existing script to perform the data extraction.    
+   
+   
+In this case, it will be necessary for the Requestor to meet with the Data Clinic team to discuss their project and its data requirements. 
 
+
+
+If a request is allocated to pathway "2", then a meeting is arranged between the Requestor and the Data Clinic team. 
+
+At this meeting, we discuss:  
+- The project aims  
+- The data requirements including inclusion/exclusion criteria for patients  
+- How the data will help answer the project aims  
+- What data is actually available on EHRS and how this may affect the project aims/scope  
+- What format the Requestor requires the data to be in (e.g. do they need an excel spreadsheet with raw data, or do they need the data processed into summary tables/graphs/etc)  
+
+
+Following this meeting, the Data Clinic team will prepare the data extraction (or a limited subset of the full extraction).  
+
+Once this is done, the Data Clinic team meet again with the Requestor to show them the data extraction.   
+-  If the Requestor is happy with this data, the Data Clinic team then proceed to a full data extraction and deliver it to the Requestor.   
+-  If the Requestor are not satisfied, the extraction is revised according to the feedback received. Further meetings are then arranged with the Requestor after each round of extraction, until the data provided is adequate.
+
+(see the below flowchart for a graphical representation of this process) 
+
+
+```{mermaid}
+flowchart TD
+A(Data request received) --> B(Screened by Data Clinic team)
+B --> C(Meeting with Data Clinic team)
+B --> K(Can be served by existing report)
+C --> D[1st extraction of data]
+D --> F{2nd meeting between requestor & Data Clinic team}
+F --> G[Happy with data]
+F --> H[Revisions needed]
+G --> I(Full extraction & data delivered)
+H --> J(Further revision to extraction)
+J --> F
+K --> L(Data delivered)
+
+linkStyle default stroke: black;
+```
+
+
+# Measurement techniques
+
+
+To collect the data on the Outcome/Process/Balancing measures above, we used the following techniques: 
+
+1) Feedback surveys sent to Requestors
+
+>  2 surveys were used:  
+>  1. "Immediate" feedback survey. This was sent to Requestors immediately after the request had been served. This focused on their experience of coming through the Data Clinic - and if they were satisfied with the process.   
+>  2. "Delayed" feedback survey. This was sent to Requestors weeks-to-months later, after the Requestors have had time to use the data for their project. This focused on their satisfaction with the data delivered - as by then they have had the experience of using the data towards their project goals, so can make a judgement on its true usefulness.   
```

Comment:
only a suggestion for phrasing: ' - by this point they will have had the opportunity to use the data for the purposes of their project...'